README.md

```diff
@@ -0,0 +1,15 @@
+<!-- prettier-ignore-start -->
```

Comment:
Can you add getting started instructions for how to set up and run this app locally?

Reply:
I've added the instructions to [Readme.md](https://github.com/ButterCMS/buttercms-marketing-site-nextjs-react/blob/bb6d7f026f7455eea8dbd6d2945eb4564c54f54c/README.md).